docs/mobile.md

```diff
@@ -0,0 +1,80 @@
+
+## Debugging with the desktop browser
+
+After you have completed the [§ Use the app from a mobile device](#use-the-app-from-a-mobile-device) instructions above, you may further want to use your desktop browser's development and dubugging tools.
+
+To do this, you will plug your phone into your laptop. You will need a USB cable. It does not work via WiFi.
+
+### Android / Chrome
+
+These instructions will allow you to debug your phone browser with Chrome DevTools on your development machine. Also, they let you view and interact with your phone's browser screen on your laptop screen or development monitor, which lets you screenshare your development work with coworkers.
+
+1. In your Android phone, turn on USB debugging. This will allow your development computer to connect to your phone.
+
+   **USB debugging** is a setting the **Developer options** menu. This menu may be hidden on your phone. It can be revealed with a ["magic tap"](https://developer.android.com/studio/debug/dev-options) on the phone **Build number** 5 times.
+
+
+2. Plug your Android phone into your development computer with a USB cable. Don't use a USB hub; cable it directly into your computer. If you see a message on your phone asking you to **Allow USB debugging** click to allow it.
+
+3. Visit `chrome://inspect` in the Chrome browser of your development computer. (It may already be open from the previous set of instructions.) You should see a listing of all the tabs open on your phone. Find the item on the list that represents the sign in screen of the identity-idp app. It should be at the top of the list.
+
+4. Click to **inspect** this tab. You should see browser DevTools and a representation of your phone's screen on your development computer, as in this illustration:
+
+    <img width="800" alt="inspect-androd-chrome-tab" src="https://user-images.githubusercontent.com/546123/231608143-aff2e115-e672-4411-8670-79f86fcf58ad.png">
+
+### iPhone / Safari
+
+These instructions work only if your development computer is an Apple product. You will need a USB cable with the appropriate "lightning" connector to plug into an iPhone.
+
+1. On your development Apple machine, open the Safari web browser. Go to menu items **Safari → Preferences → Advanced** and check **Show Develop menu in menu bar**. A screenshot:
+
+    ![show_develop_menu](https://user-images.githubusercontent.com/546123/232129916-3c68d950-1145-4af6-9a1a-c8e7c3dea7a1.png)
+
+2. Take a glance at the newly-revealed **Develop** menu item in Safari. Seeing how the menu looks now may help you find your iPhone when it later appears in the menu.
+
+3. On your iPhone, go to **Settings → Safari → Advanced** and turn on Web Inspector. Make sure JavaScript is also on.
+
+4. Plug your iPhone into your development computer with a USB cable. Don't use a USB hub; cable it directly into your computer. If you see a message on your phone asking you to **Trust This Computer?** click to trust it.
```

Comment:
Curious about the hub suggestion here. I use a USB hub and it works fine? Both for Android and iOS.

Reply:
Didn't work for me, for whatever reason. I've changed it to "may or may not work"